README.md

```diff
@@ -1,8 +1,83 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
 
+
+# Shawn(Xinyu) Wang . Jarvis Consulting
+I recently graduated from Carleton University with a Bachelor's Degree in Electrical Engineering. 
+After I graduate, I Choose to move my career path more towards the software side.
+I am passionate about learning new stuff as well as applying new skills.
+Over the last year, I have self-taught myself computer science data structures and algorithms and I have done several projects that related to programming.
```

Comment:
I am passionate about learning new stuff as well as applying new skills ~~Over the last year, I have self-taught myself computer science data structures and algorithms~~ and I have done several software development projects using [programming lanagues], [frameworks], [tools] .. 

You don't want to focus on the self-taught part